addons/html_builder/static/src/website_builder/plugins/customize_website_plugin.js

```diff
@@ -352,21 +355,40 @@ export class CustomizeWebsitePlugin extends Plugin {
                 for (const a of item.getActions()) {
                     if (a.actionId === "websiteConfig") {
                         for (const view of a.actionParam.views || []) {
-                            toDisable.add(view);
+                            if (view.startsWith("!")) {
```

Comment:
maybe you can simpify the logic somehow with a helper like this (just random thought, maybe something else would work better). not sure, but maybe worth thinking about it 
```js
const prepareView = (view, disable) => {
  let negated = view.startsWith("!");
  let set = (negated ? !disable : disable) ? toDisable : toEnable;
  set.add(negated ? view.substring(1) : view);
}
```